rules-tests/Carbon/Rector/New_/DateTimeInstanceToCarbonRector/config/configured_rule.php

```diff
@@ -4,7 +4,12 @@
 
 use Rector\Carbon\Rector\New_\DateTimeInstanceToCarbonRector;
 use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\Name\RenameClassRector;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->rule(DateTimeInstanceToCarbonRector::class);
+    $rectorConfig->ruleWithConfiguration(RenameClassRector::class, [
+            'DateTime' => 'Carbon\Carbon',
+            'DateTimeImmutable' => 'Carbon\CarbonImmutable',
+        ]);
```

Comment:
this as well, please remove, we don't want to change any class reference to avoid bc break

```php
function foo(DateTime $d)
{
}
```

see https://getrector.com/demo/47c44de7-64ef-4645-9808-0c46515c3bb6 , only on `New_` usage :)